dev/io.openliberty.jakarta.websocket.2.1/build.gradle

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+// Local vars
+File downloadSrcDir = project.file('build/source')
+
+configurations {
+    sourceJar
+}
+
+dependencies {
+     sourceJar 'jakarta.websocket:jakarta.websocket-api:2.1.0:sources'
+}
```

Comment:
Do we normally pull down sources for the API? Is this necessary? I just want us to be consistent when possible.